pkgs/unified_analytics/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
-## 5.8.6-wip
+## 6.0.0-wip
 
 - Refactored session handler class to use the last modified timestamp as the last ping value
+- Consolidate `Session` functionality into `UserProperty`
+- Remove `ErrorHandler` class and move its functionality/logic into `AnalyticsImpl`
+- Get rid of `late` variables throughout implementation class, `AnalyticsImpl`
```

Comment:
FYI (generally) no need to document user-hidden features in the changelog

Reply:
Thanks! In this case, these updates are actually related to race conditions we saw in crash reporting... i can add more detail to show how that affects users